js/app/dashboard/redux/actions.js

```diff
@@ -286,14 +290,21 @@ export function importError(token, message) {
   return { type: IMPORT_ERROR, token, message }
 }
 
-export function modifyTaskList(username, tasks) {
+export function modifyTaskList(username, tasks, updateTourUI=null) {
```

Comment:
Could you please explain why do we need to pass `updateTourUI` callback and can not execute it before dispatching this action?